### DISCUSSION TITLE
Projects are automatically disappearing 👻

### DISCUSSION THREAD

**Discussion by mansona** [community/community#10693] [Projects and Issues]
I've had a problem with creating a new project this morning, whenever I tried to make it public it seemed to just disappear 🙈 

I tried again to just add a collaborator before making it public, and it seems like as soon as I got to the access screen it might have been deleted in the backend.

Edit: it seems like you might not need to go so far as the access screen for the project to disappear 😞 

## To reproduce: 

1. Create a new project in an org https://github.com/orgs/ember-learn/projects?type=beta
2. click new project and select "New project (beta)"
3. rename project to "Ember RFC" (assuming this isn't relevant but who knows)
4. refresh and it seems to have disappeared already 😭

**Comment by Asmilex** [comment]
Same thing happening here

**Comment by mansona** [comment]
So it turns out it was an eventual consistency problem 🙈  When we were trying to create the projects we opened a bunch of them to test what was wrong (and figure out the steps for this feedback) and we ended up creating 7 extra projects that all disappeared.

When we went back to the list a few hours later (I don't know how long it actually took) then all of our projects were created 😭 

I know eventual consistency is hard but this is clearly not the greatest experience for people working with a newly created project.
